### DISCUSSION TITLE
[ 功能建议 ]添加冗余索引和没用过索引的展示功能

### DISCUSSION THREAD

**Discussion by czxin788** [hhyo/Archery#1587] [General]
### 目前遇到的问题/使用障碍
很喜欢会话管理里面的进程管理、TOP表空间和锁信息这几个功能，给运维和开发人员带来了极大便利。
最近想如果在这能看数据库冗余索引和从来没用过的索引，是不是也有使用价值。

### 希望如何解决/实现它
希望能加个查看冗余索引和从来没用过的功能。
实现方法：
select * from sys.schema_redundant_indexes \G
select * from sys.schema_unused_indexes  \G

### 其他信息
无

**Comment by LeoQuote** [comment]
欢迎直接提交pr
